recipe/meta.yaml

```diff
@@ -1,45 +1,47 @@
-{% set version = "2.17.7" %}
-{% set min_astroid = "2.15.8" %}
-{% set max_astroid = "2.17.0-dev0" %}
+{% set version = "3.0.0" %}
+{% set min_astroid = "3.0.0" %}
+{% set max_astroid = "3.1.0-dev0" %}
 
 package:
   name: pylint
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pylint/pylint-{{ version }}.tar.gz
-  sha256: f4fcac7ae74cfe36bc8451e931d8438e4a476c20314b1101c458ad0f05191fad
+  sha256: d22816c963816d7810b87afe0bdf5c80009e1078ecbb9c8f2e2a24d4430039b1
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv
```

Comment:
grayskull adds `--no-build-isolation` but does not use `--ignore-installed --no-cache-dir`. Do we care?

Reply:
sure, those other ones don't do much/anything